app/models/blueprint.rb

```diff
@@ -1,4 +1,6 @@
 class Blueprint < ApplicationRecord
+  has_many :versions, :class_name => 'Blueprint',
+           :foreign_key => :original_blueprint_id, :primary_key => :original_blueprint_id
```

Comment:
@Fryguy The purpose of the active_version column is to be able to revert to a previous version, so the last one won't always be `active`